src/stactools/nisar_sim/commands.py

```diff
@@ -53,8 +53,14 @@ def create_collection_command(destination: str) -> None:
         type=click.Choice(["129", "138", "143"]),
         help="set of numbers indicating the item's nmode ('129','138','143')",
     )
+    @click.option(
```

Comment:
We can just remove it altogether for now

Reply:
done in the new PR